packages/react/src/hooks/usePopoverPositioning.jsx

```diff
@@ -116,6 +118,12 @@ export const usePopoverPositioning = ({
         }
       }
 
+      let left = 0;
+
+      if (langDir === 'rtl' && isOverflowMenu) {
+        left = -15;
```

Comment:
Put this value 15 in a well named const that explains the origin?

Reply:
It's not necessary. Carbon has fixed it and I need to take it out in the carbon upgrade PR...

Reply:
Actually, nvm. It's not. I'll add a constant.